pkg/console/validator.go

```diff
@@ -169,6 +196,10 @@ func (v ConfigValidator) Validate(cfg *config.HarvesterConfig) error {
 		return err
 	}
 
+	if err := checkVip(cfg.Vip, cfg.VipHwAddr, cfg.VipMode); err != nil {
```

Comment:
Should only check when `cfg.Install.Mode == config.ModeCreate`

Reply:
Yes.